packages/airnode-deployer/src/cli/commands.ts

```diff
@@ -54,6 +54,13 @@ export async function deploy(configPath: string, secretsPath: string, receiptFil
     }
   }
 
+  const httpSignedRelayedGateway = config.nodeSettings.httpSignedRelayedGateway;
+  if (httpSignedRelayedGateway.enabled) {
+    if (httpSignedRelayedGateway.maxConcurrency !== undefined && httpSignedRelayedGateway.maxConcurrency <= 0) {
```

Comment:
This should be handled by validator, can you add a TODO?

Reply:
We do disallow that, at least when you're removing with the `receipt.json` as the airnode version from receipt is checked.

As for what would actually happen if you tried it, I think that it would succeed anyway. The TF state is stored in the bucket and you're doing a delete operation, so that would need to delete those resources that are actually deployed, so the ones from that state file.